ocaml/libs/tracing/tracing.mli

```diff
@@ -76,10 +78,20 @@ module Tracer : sig
   val span_is_finished : Span.t option -> bool
 
   val span_hashtbl_is_empty : unit -> bool
+
+  val finished_span_hashtbl_is_empty : unit -> bool
 end
 
 module TracerProvider : sig
-  type t
+  type t = {
+      name_label: string
+    ; tags: (string * string) list
```

Comment:
If the list is long and needs to be accessed, use a more efficient data structure.

Reply:
we're not expecting the tags list to be long